src/test/java/org/delivery/KiwiEats/repositories/ProductRepositoryTest.java

```diff
@@ -19,15 +19,15 @@ public class ProductRepositoryTest {
   @Transactional
   @Rollback
   public void testCreateProduct() {
-    Product savedProduct = productRepository.save(Product.builder()
+    Product savedProductDTO = productRepository.save(Product.builder()
```

Comment:
Is it necessary to name it as saveProductDTO?

Reply:
Intellij renamed it, since there was a problem with renaming
. I was trying to refractor and it just changed everything to DTO so I had to do it manually
.